src/CTA.Rules.Update/CodeReplacer.cs

```diff
@@ -251,7 +251,7 @@ public List<GenericActionExecution> ValidateActions(List<GenericActionExecution>
 
                 if (string.IsNullOrEmpty(actionValidation.CheckComments) || !bool.Parse(actionValidation.CheckComments))
                 {
-                    trimmedResult = Utils.EscapeAllWhitespace(root.NoComments().NormalizeWhitespace().ToFullString());
+                    trimmedResult = root.RemoveAllTrivia().ToFullString();
```

Comment:
Why are we not escaping white space anymore? As far as i know, removing trivia doesn't remove the whitespace in the code.

Reply:
Trivia include white spaces, empty lines and comments. Basically what we previously do here was trying to remove all trivia. That's also why the function name change regarding to this context.